pkg/sql/sem/builtins/builtins.go

```diff
@@ -2196,6 +2196,7 @@ CockroachDB supports the following flags:
 
 	// Metadata functions.
 
+	// https://www.postgresql.org/docs/9.6/static/functions-info.html
```

Comment:
nit: maybe better to reference the most recent docs?

Reply:
done here and below